docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
 
   postgres_bootstrap:
     image: postgres:12
-    restart: on-failure:3
+    restart: on-failure
```

Comment:
Would 'always' be appropriate for this service? https://docs.docker.com/config/containers/start-containers-automatically/#restart-policy-details

Reply:
No. This is just a bootstrap file, so it only runs once